db/models.py

```diff
@@ -50,3 +56,64 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        "MovieSession",
+        on_delete=models.CASCADE,
+        related_name="tickets",
+    )
+    order = models.ForeignKey("Order", on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def clean(self) -> None:
+        max_rows = self.movie_session.cinema_hall.rows
+        max_seats = self.movie_session.cinema_hall.seats_in_row
+
+        if not (1 <= self.row <= max_rows):
+            raise ValidationError({
+                "row": f"row number must be in available range: "
+                       f"(1, rows): (1, {max_rows})"
+            })
+        if not (1 <= self.seat <= max_seats):
+            raise ValidationError({
+                "seat": f"seat number must be in available range: "
+                        f"(1, seats_in_row): (1, {max_seats})"
+            })
```

Comment:
The `clean` method in the `Ticket` model correctly checks if the `row` and `seat` are within valid ranges. Ensure that this method is called before saving a `Ticket` instance, which is already handled by the `save` method calling `self.full_clean()`.